### PR TITLE
fix dead link to pydeck installation in readme

### PR DESCRIPTION
**Pull request by gregsadetsky**
- fixes a dead link

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ npm install deck.gl
 pip install pydeck
 ```
 
-- [Get started](https://pydeck.gl/installation.html)
+- [Get started](https://deckgl.readthedocs.io/en/latest/installation.html)
 - [Examples](https://pydeck.gl/)
```
Comment (donmccurdy) [inline docs/README.md:73]: Probably best if we're consistent about the domain – I don't have a preference on using the pydeck domain or not though. @ibgreen is this OK?


```suggestion
- [Get started](https://pydeck.gl/en/latest/installation.html)
- [Examples](https://pydeck.gl/)
```

**Review by ibgreen — APPROVED** [review]
After some complications, Linux Foundation now owns pydeck.gl so it should be fine to use it.

**Comment by ibgreen** [comment]
The link might have been dead since the pydeck domain was temporarily lost before LF acquired it. The issue might be fixed now and this PR no longer needed?

**Comment by gregsadetsky** [comment]
> The link might have been dead since the pydeck domain was temporarily lost before LF acquired it. The issue might be fixed now and this PR no longer needed?

the PR was mostly originally about fixing https://pydeck.gl/installation.html (404) and replacing it with https://pydeck.gl/en/latest/installation.html

**Pull request merged**